webcomponents/bundle/src/main/bundle/grader.properties

```diff
@@ -46,3 +48,7 @@ lti_grade_launch_instructions=The submissions for this assignment are stored in
 lti_grade_not_automatic=<b>Note:</b> When the LTI tool sends a grade to the server, it is not automatically reflected in this user interface until you refresh the grading interface.
 destroy_rubric_panel_log=Failed to destroy rubric panel. Maybe it wasn't showing in the first place.
 submission_inline=Inline Submission
+feedback_instruction = Use the box below to enter additional summary comments about this \
+submission and click Done when you've finished. You'll still need to save the grade
```

Comment:
I'd consider using another property for the shared bit, instead of repeating it all and potentially adding slight differences like the one I mentioned above